### PR TITLE
Setup basic rails app

### PR DESCRIPTION
**Pull request by christerforsgren91**
PT: https://www.pivotaltracker.com/story/show/178091740

### User stories
```
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.
```
### Changes proposed
- Basic scaffolded application
- Cors configurations

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+group :development do
+  gem 'listen', '~> 3.3'
+  gem 'spring'
+end
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (emtalen) [inline Gemfile]: remove this, it's for windows

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
```
Comment (emtalen) [inline config/initializers/cors.rb]: please remove these comments. If you visit a file and change something then clean the comments

**Review by emtalen — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (christerforsgren91) [reply Gemfile]: gem is removed!

File: config/initializers/cors.rb (hunk 2 above)
Comment (christerforsgren91) [reply config/initializers/cors.rb]: yes. the comments are cleaned up.

**Review by emtalen — APPROVED** [review]

**Pull request merged**